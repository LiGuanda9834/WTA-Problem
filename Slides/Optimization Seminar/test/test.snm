\beamer@slide {Fig.main2}{7}
