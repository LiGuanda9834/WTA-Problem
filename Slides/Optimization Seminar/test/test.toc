\beamer@sectionintoc {1}{Introduction}{3}{0}{1}
\beamer@sectionintoc {2}{Formulations and Algorithms}{10}{0}{2}
\beamer@sectionintoc {3}{Outer Approximation}{17}{0}{3}
\beamer@sectionintoc {4}{Columnn Generation}{29}{0}{4}
\beamer@sectionintoc {5}{Future Work}{40}{0}{5}
