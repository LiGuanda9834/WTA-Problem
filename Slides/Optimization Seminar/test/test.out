\BOOKMARK [2][]{Outline0.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [2][]{Outline0.2}{\376\377\000F\000o\000r\000m\000u\000l\000a\000t\000i\000o\000n\000s\000\040\000a\000n\000d\000\040\000A\000l\000g\000o\000r\000i\000t\000h\000m\000s}{}% 2
\BOOKMARK [2][]{Outline0.3}{\376\377\000O\000u\000t\000e\000r\000\040\000A\000p\000p\000r\000o\000x\000i\000m\000a\000t\000i\000o\000n}{}% 3
\BOOKMARK [2][]{Outline0.4}{\376\377\000C\000o\000l\000u\000m\000n\000n\000\040\000G\000e\000n\000e\000r\000a\000t\000i\000o\000n}{}% 4
\BOOKMARK [2][]{Outline0.5}{\376\377\000F\000u\000t\000u\000r\000e\000\040\000W\000o\000r\000k}{}% 5
